angel_system/activity_hmm/core.py

```diff
@@ -236,7 +236,7 @@ def __init__(self, dt, class_str, med_class_duration,
         class_str_.append('background')
         class_str_map.append(len(class_str_) - 1)
         bckg_mask.append(True)
-        inv_map.append(i)
+        inv_map.append(0)
```

Comment:
Good you fixed this. I made another PR for this, but I'll just let it be fixed here.

Reply:
Also in your other branch, so we'll see who gets in first and then the other will likely need to rebase. :)